apps/projects/package.json

```diff
@@ -29,8 +29,6 @@
     "test": "TRUFFLE_TEST=true npm run ganache-cli:test"
   },
   "dependencies": {
-    "@aragon/api": "1.1.0",
-    "@aragon/api-react": "1.0.0-beta.2",
```

Comment:
I think it would be a good chance to add the new versions of `api` and `api-react` to every package using them, that will ease the integrations once merged to dev